pom.xml

```diff
@@ -49,6 +49,11 @@
     <dependencies>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>display-url-api</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
:-1: vote from me

Snapshot dependencies **shouldn't** _ever be proposed_ as a change to a stable plugin.  Stable release dependences only, please.  Snapshots are good for on-the-fly development but not for contributing changes which would make it to a "stable" release.  The build for this plugin would no longer be stable because it's built against a moving target; a snapshot.